docs/tutorials/todo-list-app/process-a-new-item.md

```diff
@@ -50,14 +50,15 @@ namespace ToDoList.ViewModels
                 addItemViewModel.OkCommand,
                 addItemViewModel.CancelCommand.Select(_ => (ToDoItem?)null))
                 .Take(1)
-                .Subscribe(newItem =>
+                .Subscribe(Observer.Create<ToDoItem?>(newItem =>
```

Comment:
For ReactiveUI there is no difference between 0.10 and 11.0 here. In both cases you need to add "using System;".

So, it might need to be added here instead.